archiver.rb

```diff
@@ -0,0 +1,38 @@
+require 'tempfile'
+require 'fileutils'
+require 'find'
+
+class Archiver
+  attr_accessor :file
+  @modes = %i[reader writer both]
+
+  def initialize
```

Comment:
You could delete the empty constructor `def initialize` with any issue

Reply:
it can be also initializated like `def initialize; end`.